_includes/scenarios/integrate/propose-attribute-to-peer.md

```diff
@@ -1,5 +1,215 @@
+{: .notice--info}
+
+<div style="width: 640px; height: 480px; margin: 10px; position: relative;"><iframe allowfullscreen frameborder="0" style="width:640px; height:480px" src="https://lucid.app/documents/embedded/25159913-8e8c-4f65-98eb-f2522b8248a6" id="zk3XNeKG5NpM"></iframe></div>
+
+### Accept a ProposeAttributeRequestItem
+
+If the Recipient wants to give the Sender feedback on the validness of one of the Attributes proposed by the Sender, it can accept the associated ProposeAttributeRequestItem contained in the Request for proposing Attributes using appropriate [AcceptProposeAttributeRequestItemParameters]({% link _docs_integrate/data-model-overview.md %}#acceptproposeattributerequestitemparameters). Depending on whether the Recipient confirms the validness of the proposed Attribute, it can agree to its creation and send it back to the Sender unchanged or correct the Attribute Value beforehand. This means that the Recipient needs to specify the proposed Attribute or an Attribute that differs from the proposed Attribute only by the Attribute Value as the value for the `attribute` parameter of the AcceptProposeAttributeRequestItemParameters. If the Recipient does not want to create a new Attribute, it can alternatively send an existing Attribute back to the Sender. The `attributeId` parameter of the AcceptProposeAttributeRequestItemParameters must be used for that. The existing Attribute may only differ from the proposed Attribute by the Attribute Value. The acceptance of a ProposeAttributeRequestItem leads to the creation of a corresponding [LocalAttribute]({% link _docs_integrate/data-model-overview.md %}#localattribute) with a [LocalAttributeShareInfo]({% link _docs_integrate/data-model-overview.md %}#localattributeshareinfo) contained within its `shareInfo` property. The `content` of the LocalAttribute is the Attribute that the Recipient wants to send back to the Sender. Depending on whether the Recipient confirms the validness of the proposed Attribute, it is therefore the proposed Attribute itself or a corrected version of it. Based on this LocalAttribute, an appropriate AcceptResponseItem of type [ProposeAttributeAcceptResponseItem]({% link _docs_integrate/data-model-overview.md %}#proposeattributeacceptresponseitem) is generated, which incorporates the `id` of the LocalAttribute with the LocalAttributeShareInfo in its `attributeId` property and the Attribute that the Recipient wants to send back to the Sender in its `attribute` property. This ProposeAttributeAcceptResponseItem will be contained within the `items` property of the [Response]({% link _docs_integrate/data-model-overview.md %}#response) to the Request for proposing Attributes that will be transferred to the Sender. If a new IdentityAttribute is to be created and sent back to the Sender, a corresponding LocalAttribute without a LocalAttributeShareInfo is additionally created for the Recipient beforehand.
```

Comment:
I somehow stumble upon the first sentence. Also, in the case of a proposed subscription to a newsletter, it doesn't seem to fully apply. Maybe reformulate the first part of that sentence.

Also the second sentence.

The third sentence is a bit complex, too. I'd propose to split it up in two sentences for better understanding.

Later you talk again about correcting and validness. Perhaps you can formulate it more generically. And, if you'd like to reference "validness", I'd use the term "validity", which I feel is used much more commonly.